tests/testthat/test-gaussian.R

```diff
@@ -0,0 +1,20 @@
+context("gaussian regression")
```

Comment:
You can do more tests here based on what we know about the solution for the lasso. At a minimum, I'd test: 

i) lambda = 0 ==> get back OLS solution
ii) lambda > lambda_max ==> get back all zero solution
iii) the correct solution conditional on knowing the active set
iv) Closed form solution for ridge regression

Ping me if you need help devising these tests

Reply:
I agree, this should be added too.

Reply:
agree